spec.html

```diff
@@ -25936,44 +25936,44 @@
 
       <!-- es6num="20.3.1.7" -->
       <emu-clause id="sec-local-time-zone-adjustment">
-        <h1>Local Time Zone Adjustment</h1>
-        <p>An implementation of ECMAScript is expected to determine the local time zone adjustment. The local time zone adjustment is a value <dfn>LocalTZA</dfn> measured in milliseconds which when added to UTC represents the local <em>standard</em> time. Daylight saving time is <em>not</em> reflected by LocalTZA.</p>
+        <h1>LocalTimeZoneAdjustment ( _t_, _isUTC_ )</h1>
```

Comment:
I do prefer 'offset' to 'adjustment', too. That's what I did in my earlier PR. However, the sign of Date.prototype.getTimezoneOffset is exactly the opposite of what's more common. That is, getTimeZoneOffset() returns a negative value for west of the primary meridian and a positive value for east of the primary meridian. 

I can use "offset' and match its sign with that of getTimeZoneOffset(), but I find it rather confusing when I talk about 'positive' offset change or 'negative' offset change. It's still a possibility, but I chose otherwise by sticking to the existing term in 'Local Timezone Adjustment'.